.gitlab/hpsf-gitlab-ci.yml

```diff
@@ -133,7 +133,7 @@ AMD-MI300A:
 NVIDIA-GRACE-GRACE:
   stage: test
   tags: [uo, aarch64, cpu, grace, neoverse_v2]
-  image: nvidia/cuda:12.6.1-devel-ubuntu24.04
+  image: tcclevenger/kokkos-ubuntu:24.04
```

Comment:
hmm I gues we want to avoid custom images, see https://github.com/kokkos/kokkos/pull/7715